themes/blist/ukrainerInTrier/content/en/blog/need-help.md

```diff
@@ -0,0 +1,43 @@
+---
+author: 'Hugo Authors'
+title: 'I need help'
+date: 2021-07-15
```

Comment:
Form pages should not be like blog pages. They should be separate independent pages. No date or author.

Reply:
+